app/Containers/Authentication/composer.json

```diff
@@ -2,7 +2,7 @@
   "name": "apiato/authentication",
   "description": "apiato Container.",
   "require": {
-    "laravel/passport": "2.*"
```

Comment:
this supposed to be a different commit by itself, in case we want to revert it or test the entire app after this change.